kratos/python/add_utilities_to_python.cpp

```diff
@@ -285,6 +356,81 @@ void CopyModelPartNodalVarToNonHistoricalVarWithDestination(
     rVariableUtils.CopyModelPartNodalVarToNonHistoricalVar(rVariable, rDestinationVariable, rOriginModelPart, rDestinationModelPart, BuffStep);
 }
 
+template<class TVarType>
+void SetScalarVarWithoutBuffer(
+    VariableUtils &rVariableUtils,
+    const TVarType& rVariable,
+    const double Value,
+    NodesContainerType& rNodes
+    )
+{
+    rVariableUtils.SetScalarVar(rVariable, Value, rNodes);
+}
+
+template<class TVarType>
+void SetScalarVarWithBuffer(
+    VariableUtils &rVariableUtils,
+    const TVarType& rVariable,
+    const double Value,
+    NodesContainerType& rNodes,
+    const IndexType BuffStep
+    )
+{
+    rVariableUtils.SetScalarVar(rVariable, Value, rNodes, BuffStep);
+}
+
+template<class TVarType>
+void SetScalarVarForFlagWithoutBufferNotCustomCheck(
```

Comment:
What is the name means?

Reply:
By default the flag is set as true, but with this you can set as true or false

Reply:
So wouldn't be better to have it as a default value in the other one?

Reply:
Yes, the problem is that these method gave errors when exporting to python and later execute them

Reply:
Why? Would you please describe more?

Reply:
At least what I saw, pybind may give problem when exporting to python methods with default values. When you try to execute them it says that the arguments are wrong

Reply:
I see, but then you may use two different helper functions like overloaded methods. Isn't it?

Reply:
It is what I am doing, but with different names

Reply:
just FYI: pybind cannot extract defaults from c++, but they can still be specified (separately)
See e.g. https://github.com/KratosMultiphysics/Kratos/blob/a1ca9e7d0fd006539f9605332a8a9444ad085cba/applications/CoSimulationApplication/custom_python/add_custom_io_to_python.cpp#L358

Reply:
Done, thanks

Reply:
@pooyan-dadvand I removed those duplicated methods